packages/react/test/utils/withProvider.tsx

```diff
@@ -20,18 +20,30 @@ export const EmptyThemeProvider: React.FunctionComponent = ({ children }) => {
   return <ThemeProvider theme={theme}>{children}</ThemeProvider>
 }
 
-export const mountWithProvider = (node, options?, theme?: ThemeInput) => {
+interface AugmentedMountRendererProps extends MountRendererProps {
+  wrappingComponent?: React.FunctionComponent
+}
```

Comment:
Can we just update types for `enzyme`?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f44f31789c08ce190cf58b0d441977dabb968f24/types/enzyme/index.d.ts#L700

Reply:
I updated the types but I still need to put `as any` in `setProps`. With these changes I don't need to do that.